src/ui/public/registry/field_formats.js

```diff
@@ -86,6 +86,7 @@ class FieldFormatRegistry extends IndexedArray {
    */
   getInstance = _.memoize(function (formatId) {
     const FieldFormat = this.byId[formatId];
+    if (!FieldFormat) throw new Error(`Field Format '${formatId}' not found!`);
```

Comment:
Missing braces here, needs to follow [elastic js styleguide](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops)